test/jdk/java/util/Locale/LegacyCodesClassInvariant.java

```diff
@@ -93,16 +89,16 @@ private void verify(String lang) {
                 final Locale loc = (Locale)in.readObject();
                 final Locale expected = Locale.of(lang, "XX");
                 if (!(expected.equals(loc))) {
-                    errln("Locale didn't maintain invariants for: "+lang);
-                    errln("         got: "+loc);
-                    errln("    excpeted: "+expected);
+                    fail("Locale didn't maintain invariants for: "+lang);
+                    fail("         got: "+loc);
+                    fail("    excpeted: "+expected);
```

Comment:
This will log the message in the first `fail() invocation. This may be auto-converted by the script, so you might want to check other places.

Reply:
Good point, adjusted all locations of this to be contained in one `fail()` call.